### DISCUSSION TITLE
Get album by barcode returns 404

### DISCUSSION THREAD

**Discussion by anatosun** [tidal-music/discussions#41] [Report bug]
When querying the API endpoint `/albums/byBarcodeId`, I unfortunately get a 404 error, although the barcode exists.

- Here is the query:
```bash
curl -X 'GET' \
  'https://api.tidalhifi.com/v1/albums/byBarcodeId?barcodeId=00854242007552&countryCode=US' \
  -H 'accept: application/vnd.tidal.v1+json' \
  -H 'Authorization: Bearer redacted' \
  -H 'Content-Type: application/vnd.tidal.v1+json'
```
-  and the error
```json
{"status":404,"subStatus":2001,"userMessage":"Resource not found"}
```

**Comment by stefanmoro** [comment]
Hey @anatosun,

I believe you are calling the wrong domain. Please refer to the [TIDAL API reference](https://developer.tidal.com/apiref?spec=catalogue&ref=get-albums-by-barcode-id) for the `albums/byBarcodeId` endpoint for correct domain.

**Comment by anatosun** [comment]
Thank you for your answer. However, whenever I try to call the openapi endpoint with:

```bash
curl -X 'GET' \
  'https://openapi.tidal.com/albums/byBarcodeId?barcodeId=00854242007552&countryCode=US' \
  -H 'accept: application/vnd.tidal.v1+json' \
  -H 'Authorization: Bearer redacted' \
  -H 'Content-Type: application/vnd.tidal.v1+json'
  ```

I get:
```json
{"errors":[{"category":"API_ERROR","code":"SERVICE_UNAVAILABLE","detail":"The service is currently unable to handle requests"}]}
```

**Reply by aliaksei-taliuk** [reply]
Hi @anatosun 

The credentials you’re using represents an unauthorized usage of our platform. You should sign up for the Developer Portal and register a new Open API application and use these creds instead. You can find more information in our [documentation](https://developer.tidal.com/documentation/api/api-quick-start).
